docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/03_LORIS_MRI_Install/Signal_Viewer_UI_Install.md

```diff
@@ -0,0 +1,277 @@
+
+
+# Signal Viewer UI Install
+
+> Last update: 2022-11-02
+
+## Context
+
+This document is a guide to install and configure Loris HBCD to use the `Signal Viewer` UI for EEG (BIDS files).
+
+## TOC
+
+1. Install
+    1. [Environment](#environment)
+    1. [Versions](#versions)
+    1. [Install Loris-MRI](#install-loris-mri)
+    1. [Install Protobuf](#install-protobuf)
+    1. [Update `electrophysiology_browser` module](#update-electrophysiology_browser-module)
+    1. [Installed environment](#installed-environment)
+1. [EEG upload to Loris](#eeg-upload-to-loris)
+    1. [Get EEG files](#get-eeg-files)
+    1. [Chunk EEG files](#chunk-eeg-files)
+    1. [Import BIDS to Loris](#import-bids-to-loris)
+1. [Troubleshooting](#troubleshooting)
+    1. [BIDS import](#bids-import)
+
+## Install
+
+### Environment
+
+> Note: right now, only `24.0 release` with `php 8.0` works. More recent versions are on the way.
+
+Environment:
+- `Ubuntu` machine
```

Comment:
Not mandatory, I expect it to work on our regretted CentOS. We can remove the line.